apex_rostest/README.md

```diff
@@ -0,0 +1,129 @@
+# apex_launchtest
```

Comment:
Just as a reminder for our future selves, we need to find a new home for this repo level README.